subsys/ieee802154/ieee802154_nrf5_cfg.c

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <init.h>
+#include <irq.h>
+#include <kernel.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+#include "nrf_802154.h"
```

Comment:
Please use angle brackets for non-local includes.

Reply:
fixed